.github/workflows/backport.yml

```diff
@@ -3,6 +3,11 @@ on:
   issue_comment:
     types: [created]
 
+permissions:
+  contents: write
+  issues: write
```

Comment:
Need `issues` permission to write comments into the original PR (where a user started this workflow)